config/config.yaml

```diff
@@ -6,6 +6,8 @@ launchtask:
    retryinterval: 10000
    timeout: 500000
    skippull: true
+   disabletrace: false
```

Comment:
composetrace?

Reply:
by default, we wanna enable compose trace